src/AaveV3Update_20231002matic/AaveV3Polygon_20231002matic.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IEngine,EngineFlags,Rates} from 'aave-helpers/v3-config-engine/AaveV3PayloadBase.sol';
+import {
+  AaveV3PayloadPolygon,
+  AaveV3PolygonAssets
+} from 'aave-helpers/v3-config-engine/AaveV3PayloadPolygon.sol';
+
+/**
+  * @title Gauntlet Recommendations to Lower stMATIC/MaticX non-emode LT/LTV on Polygon v3
+  * @author Gauntlet
+  * - Discussion: https://governance.aave.com/t/arfc-gauntlet-recommendation-to-lower-stmatic-maticx-non-emode-lt/14859
```

Comment:
we usually recommend to also link the snapshot, if it exists: https://snapshot.org/#/aave.eth/proposal/0x151df3127d187c203ff3a975d8ab9e34493d77edf67ec6884dc086d5aea9ce89